src/main/java/reposense/git/GitBlame.java

```diff
@@ -68,21 +85,41 @@ public static GitBlameLineInfo blameLine(String root, String commitHash, String
 
         String blameResult = StringsUtil.filterText(runCommand(rootPath, blameCommand),
                 COMBINATION_WITH_COMMIT_TIME_REGEX);
+        String[] blameResultLines = StringsUtil.NEWLINE.split(blameResult);
+        return processGitBlameResultLine(blameResultLines, "committer");
+    }
 
-        return processGitBlameResultLine(blameResult);
+    /**
+     * Returns the processed result of {@code blameResults}.
+     */
+    private static List<GitBlameLineInfo> processGitBlameResultLines(String blameResults) {
+        String[] blameResultsLines = StringsUtil.NEWLINE.split(blameResults);
+        List<GitBlameLineInfo> blameFileResult = new ArrayList<>();
+        for (int lineCount = 0; lineCount < blameResultsLines.length; lineCount += BLAME_LINE_INFO_ROW_COUNT) {
+            String[] blameResultLines = Arrays
+                    .copyOfRange(blameResultsLines, lineCount, lineCount + BLAME_LINE_INFO_ROW_COUNT - 1);
+            GitBlameLineInfo blameLineInfo = processGitBlameResultLine(blameResultLines, "author");
+            blameFileResult.add(blameLineInfo);
+        }
+        return blameFileResult;
     }
 
     /**
-     * Returns the processed result of {@code blameResult}.
+     * Returns the processed result of {@code blameResultLines}, with reference to {@code timeOption}.
      */
-    private static GitBlameLineInfo processGitBlameResultLine(String blameResult) {
-        String[] blameResultLines = StringsUtil.NEWLINE.split(blameResult);
+    private static GitBlameLineInfo processGitBlameResultLine(String[] blameResultLines, String timeOption) {
```

Comment:
I'd prefer to use a boolean for timeOption since it doesn't seem like there are likely to be new options in git in the future.